bot/exts/moderation/clean.py

```diff
@@ -531,11 +537,21 @@ async def clean_regex(
 
         `message_or_time` can be either a message to stop at (exclusive), a timedelta for max message age, or an ISO
         datetime.
-        If a message is specified, `channels` cannot be specified.
 
-        The pattern must be provided enclosed in backticks.
-        If the pattern contains spaces, it still needs to be enclosed in double quotes on top of that.
-        For example: `[0-9]`
+        If a message is specified the cleanup will be limited to the channel the message is in.
+
+        If a timedelta or an ISO datetime is specified, `channels` can be specified to clean across multiple channels.
+        An asterisk can also be used to designate cleanup across all channels.
+
+        The `regex` pattern must be provided enclosed in backticks, that will make it appear as a code section.
+
+        For example: \u02CB[0-9]\u02CB, which should appear as `[0-9]`.
+
+        If the `regex` pattern contains spaces, it still needs to be enclosed in double quotes on top of that.
+
+        For example: "\u02CB[0-9]\u02CB", which should appear as "`[0-9]`".
+
+        Do not copy and paste the backticks from the examples, they are special unicode characters that will not work.
```

Comment:
I think this bit is a bit wordier than it needs to be, I think something like this would suffice (note backticks can be escaped with `\`):
```
        The `regex` pattern must be enclosed in backticks (e.g. \`[0-9]\`). 
        If it contains a space it must also be enclosed in double quotes (e.g. "\`test [0-9]\`").
```
![image](https://user-images.githubusercontent.com/22353562/170764939-ad497e89-146c-4050-9ac9-9d719ed5c376.png)

I'm not exactly sure what change the original issue was proposing to this part though. Thoughts?

Reply:
The problem is that the intent seems to be to show how to write the regex part of the command, e.g. must be enclosed in backticks `[0-9]`, but the problem is when it appears on discord it just turns into a code block due to the [0-9] being enclosed in ``, potentially making a less knowledgeable user confused.

When I tried escaping the standard backticks with backslash the linter didn't seem to like it, so I just chose to use special unicode characters that won't turn into a code block, even though it may cause confusion if a user tries to copypaste those backticks.

Reply:
Ah, in that case you can escape the backticks explicitly to make the linter happy
```
        The `regex` pattern must be enclosed in backticks (e.g. \\`[0-9]\\`).
        If it contains a space it must also be enclosed in double quotes (e.g. "\\`test [0-9]\\`").
```
(is the same as my screenshot above)

Reply:
Yeah that seemed to do the trick, thanks! I removed some of the excessive words, so it should be a bit cleaner now.